homeassistant/components/zwave_js/__init__.py

```diff
@@ -134,6 +148,31 @@ def async_on_node_removed(node: ZwaveNode) -> None:
         # note: removal of entity registry is handled by core
         dev_reg.async_remove_device(device.id)
 
+    @callback
+    def async_on_value_notification(notification: ValueNotification) -> None:
+        """Relay stateless value notification events from Z-Wave nodes to hass."""
+        device = dev_reg.async_get_device({get_device_id(client, notification.node)})
+        value = notification.value
+        if notification.metadata.states:
+            value = notification.metadata.states.get(str(value), value)
+        hass.bus.async_fire(
+            ZWAVE_JS_EVENT,
+            {
+                ATTR_TYPE: "value_notification",
+                CONF_DOMAIN: DOMAIN,
+                ATTR_NODE_ID: notification.node.node_id,
+                ATTR_HOME_ID: client.driver.controller.home_id,
+                ATTR_ENDPOINT: notification.endpoint,
+                CONF_DEVICE_ID: device.id,
```

Comment:
Same here.

Reply:
In that case, zha integration is cheating ;-)
I'll adjust

Reply:
fixed